cvat/apps/engine/views.py

```diff
@@ -2256,24 +2284,29 @@ def _import_annotations(request, rq_id, rq_func, db_obj, format_name,
                     filename=filename,
                     key=key,
                     request=request,
+                    result_ttl=settings.IMPORT_CACHE_SUCCESS_TTL,
+                    failure_ttl=settings.IMPORT_CACHE_FAILED_TTL
                 )
 
         av_scan_paths(filename)
         meta = {
             'tmp_file': filename,
-            'tmp_file_descriptor': fd,
         }
         rq_job = queue.enqueue_call(
-            func=rq_func,
-            args=(db_obj.pk, filename, format_name, conv_mask_to_poly),
+            func=import_resource_with_clean_up_after,
```

Comment:
Probably, this function can be used as a decorator on the functions sent to the queue, this may simplify the code.

Reply:
That's interesting, I didn't know RQ doesn't support decorators. Ok, then there is no other way.